src/components/delegates/delegates.less

```diff
@@ -34,7 +34,7 @@ delegates {
     background-color: rgb(255, 228, 220);
   }
   .pending {
-    background-color: #eaeae9;
+    background-color: rgb(204, 231, 236);
```

Comment:
> Highlight voted delegate row in blue (similarly to currently used red/green)

This was supposed to mean something else. Sorry that my description was ambiguous. Pending votes should stay gray, but not pending placed votes should be blue. That is the state when the checkbox is checked and (currently) the background is white. 

Also, I personally find the blue you chose bit too greenish, I like more something like #d6f0ff